### PR TITLE
[Docs] install

### PR DESCRIPTION
**Pull request by Harold-lkk**

### CODE REVIEW

File: docs/en/get_started/install.md [hunk 1]
```diff
@@ -111,32 +113,10 @@ We provide two options to verify the installation via inference demo, depending
...
-
-The downloading will take several seconds or more, depending on your network environment. The directory tree should look like the following once everything is done:
-
-```bash
-├── crnn_academic-a723a1c5.pth
-├── crnn_academic_dataset.py
-├── dbnet_r18_fpnc_1200e_icdar2015.py
-├── dbnet_r18_fpnc_sbn_1200e_icdar2015_20210329-ba3ab597.pth
-└── demo_text_det.jpg
+python mmocr/ocr.py --det DB_r18 --recog CRNN demo/demo_text_det.jpg --show
```
Comment (Harold-lkk) [inline docs/en/get_started/install.md:119]:
```suggestion
python mmocr/ocr.py --det DB_r18 --recog CRNN demo/demo_text_ocr.jpg --show
```

File: docs/zh_cn/get_started/install.md [hunk 2]
```diff
@@ -108,32 +110,10 @@ pip install albumentations>=1.1.0 --no-binary qudida,albumentations
...
-
-取决于你的网络环境，下载过程可能会持续几十秒或者更长。一切就绪后，当前目录树应当包含以下文件：
-
-```bash
-├── crnn_academic-a723a1c5.pth
-├── crnn_academic_dataset.py
-├── dbnet_r18_fpnc_1200e_icdar2015.py
-├── dbnet_r18_fpnc_sbn_1200e_icdar2015_20210329-ba3ab597.pth
-└── demo_text_det.jpg
+python mmocr/ocr.py --det DB_r18 --recog CRNN demo/demo_text_det.jpg --show
```
Comment (Harold-lkk) [inline docs/zh_cn/get_started/install.md:116]:
```suggestion
python mmocr/ocr.py --det DB_r18 --recog CRNN demo/demo_text_ocr.jpg --show
```

File: docs/en/get_started/install.md [hunk 3]
```diff
@@ -111,40 +113,18 @@ We provide two options to verify the installation via inference demo, depending
...
-├── crnn_academic-a723a1c5.pth
-├── crnn_academic_dataset.py
-├── dbnet_r18_fpnc_1200e_icdar2015.py
-├── dbnet_r18_fpnc_sbn_1200e_icdar2015_20210329-ba3ab597.pth
-└── demo_text_det.jpg
+python mmocr/ocr.py --det DB_r18 --recog CRNN demo/demo_text_ocr.jpg --show
 ```
 
-**Step 2.** Run the following codes in your Python interpreter:
+Also can run the following codes in your Python interpreter:
```
Comment (Harold-lkk) [inline docs/en/get_started/install.md:122]:
```suggestion
You can also run the following codes in your Python interpreter:
```

File: docs/en/get_started/install.md [hunk 4]
```diff
@@ -99,10 +101,10 @@ to ['albumentations`'s official documentation](https://albumentations.ai/docs/ge
 
 ### Verify the installation
 
-We provide two options to verify the installation via inference demo, depending on your installation method. You should be able to see a pop-up image and the inference result upon successful verification.
+We provide a method to verify the installation via inference demo, depending on your installation method. You should be able to see a pop-up image and the inference result upon successful verification.
 
 <div align="center">
-    <img src="https://raw.githubusercontent.com/open-mmlab/mmocr/main/resources/verification.png"/><br>
+    <img src="https://user-images.githubusercontent.com/24622904/187707213-2d49611f-fd46-46ff-b5d9-a3e5d1979a48.png"/><br>
```
Comment (gaotongxiao) [inline docs/en/get_started/install.md:107]: fix the inference result

File: requirements/mminstall.txt [hunk 5]
```diff
@@ -1,2 +1,3 @@
-mmcv-full>==2.0.0rc0,<2.1.0
+mmcv>==2.0.0rc1,<2.1.0
 mmdet>=3.0.0rc0,<3.1.0
+mmengine>= 0.5.0
```
Comment (xinke-wang) [inline requirements/mminstall.txt:3]: https://github.com/open-mmlab/mmengine/blob/main/mmengine/version.py it seems the version has been changed to 0.1.0?

**Pull request merged**